_posts/2018-11-15-Rust-survey-2018.md

```diff
@@ -0,0 +1,168 @@
+
+# Challenges
+
+As we've done in past years, we asked for your comments in where Rust can improve. This year, we see some familiar themes as well as some new ones in this feedback. The top ten themes this year are:
+
+1. the need for better library support
+2. a more improved IDE experience
+3. the need for broader adoption of Rust generally
+4. a richer ecosystem of tools and support
+5. an improved learning curve
+6. the need for important language features and crates to stable and supported
+7. support for async programming
+8. support for GUI development
+9. better documentation
+10. improved compile times
+
+New this year is the rising need to **support GUI development**, showing that Rust continues to grow not only on the server, but that people are feeling the need to stretch into application development.
+
+> "Improve Rust marketing. Many people don't know about it"
+
+Comments remind us that while Rust may be well-known in some circles, it still has room to grow and in many tech circles Rust may not yet be well-known. 
+
+> "Keeping a focus on adoption/tutorials/books/novice experience will pay dividends in the years to come."
+
+In addition to outreach, a broader set of documentation would in turn help reach a broader audience. 
+
+> "Stability and maturity of developer tools, make it easier to get a working setup and to debug applications"
+
+Many people commented on the IDE support, pointing out not only instability or inaccuracy in the RLS, but also the need for a much stronger IDE story that covered more areas, like easier debugging.
+
+> "The maturity of the ecosystem and libraries. Have a good ecosystem of "standard" libraries is key for the future of the language"
+
+A common theme continues to be the need to push libraries to completion and grow the set of "standard" libraries that users can use. Some comments point out this isn't the fault of maintainers, who are already working hard to write and publish the crates, but that generally more companies need to get involved and offer commercial support.
+
+> "Ergonomics and discoverability of "putting it together" documentation"
+
+Some people pointed out that ergonomics goes hand in hand with richer documentation, seeing that these aren't separate concepts but rather challenges that should be tackled together in a unified approach. 
+
+## Looking forward
+
+This year saw the strongest survey yet. Not only was it the largest community survey, it was the first to cover languages outside of English. Rust continues to grow steadily, and with it, both its strengths and challenges are introduced to a broader audience. 
+
+We look forward to using your feedback in planning for 2019, and excited to see where we can take Rust next.
```

Comment:
"We look forward to using your feedback in planning for 2019, and excited to see where we can take Rust next." sounds like it's missing a word, either remove the comma after 2019 and change it to "and **are** excited" or leave the comma and change it to "and **we're** excited"?